src/pages/home/report/ReportActionItem.js

```diff
@@ -768,6 +772,10 @@ export default compose(
         userWallet: {
             key: ONYXKEYS.USER_WALLET,
         },
+        parentReportActions: {
```

Comment:
For a little extra context, I did look into subscribing to this higher up in the component tree, but it got pretty convoluted and went all the way up to `ReportScreen`. Interestingly enough, I have [this PR](https://github.com/Expensify/App/pull/33970) where `ReportScreen` is now subscribing to the parent report action... so if we really wanted to, we could pass the `parentReportAction` all the way down as a prop. It would be a bigger change, but it might have less of a possible performance impact on this component (which looks to be highly optimized).